src/client/components/TablePaginated/Filters/Filters.tsx

```diff
@@ -7,12 +7,16 @@ import Icon from 'client/components/Icon'
 import Text from 'client/components/TablePaginated/Filters/Text/Text'
 import { TablePaginatedFilter } from 'client/components/TablePaginated/types'
 
-const componentsByFilterType: Record<TablePaginatedFilterType, React.FC<TablePaginatedFilter & { path: string }>> = {
+const componentsByFilterType: Record<
+  TablePaginatedFilterType,
+  React.FC<TablePaginatedFilter<TablePaginatedFilterType> & { path: string }>
+> = {
   [TablePaginatedFilterType.TEXT]: Text,
+  [TablePaginatedFilterType.SWITCH]: () => null,
```

Comment:
We can implement this once we need to add a visible boolean filter. For now it's not necessary because the "administrators" filter is hidden.